### ISSUE TITLE
Suggestion: Hurt Command "PreventImmunity" option/ ignore invincibility frames

### ISSUE THREAD

**Issue by Rickytehduck** [DenizenScript/Denizen-For-Bukkit#1535]
As the hurt command currently works, it respects player/mob invincibility frames/won't hurt players that have recently been hurt by the command. If I run two simultaneous/close hurt commands, while both damage events are triggered, and the end result is not what you would expect.

For example, if I have an instant/speed: 0 task setup like so:
`TestHurt:
  type: task
  speed: 0
  script:
    - hurt <player> a:9 source:<npc> cause:entity_attack
    - hurt <player> a:10 source:<npc> cause:projectile`

You'd probably expect one of two results- either the player would be hurt for 19 total damage, or that they would be hurt for 9 damage, and then not be hurt by the second hurt command. 

In fact, it seems like whichever hurt command has the higher value, in this case the second one, takes priority. 

I'd really love a method to override this, or a way for damage to be calculated for both hurt commands at the same time.

TL;DR,
Currently as it stands, you can't have multiple hurt commands hitting players at once, as it prioritizes the command that is doing more damage.
I'm suggesting a change so that there is an option to either ignore existing hit cooldowns, or a way to combine the final damage? 
Preferably the first option since that will allow for multiple damage causes to hit an entity at  once, which is what I've been trying to do.

Thanks for your time.

**Comment by mcmonkey4eva** [comment]
Send a hurt command without source/cause to "Enforce" the damage, IE it's guaranteed to go through without anything being able to block it.
Include source/cause to not do that: IE it goes through Bukkit and Minecraft's safe normal proper damage methods.

**Comment by Rickytehduck** [comment]
Right, so there's no method to both have a cause AND also ignore blocking?
I'll have to work around that. Thanks for the clarification.

**Comment by Rickytehduck** [comment]
Right, so there's no method to both have a cause AND also ignore blocking?
I'll have to work around that. Thanks for the clarification.
